images/scripts/debian.setup

```diff
@@ -153,6 +157,9 @@ if [ "$IMAGE" = "debian-testing" ]; then
     COCKPIT_DEPS="${COCKPIT_DEPS/libpcp3 /}"
     COCKPIT_DEPS="${COCKPIT_DEPS/pcp /}"
     COCKPIT_DEPS="${COCKPIT_DEPS/python3-pcp /}"
+
+    # https://bugs.debian.org/XXX
```

Comment:
Filed: https://bugs.debian.org/1098793

You already triggered tests here on 913b18dced11f7fbcaaf3c69cbaaac0ded5bfd16 and they are green. So I'd say  push the comment update without re-running them again.

Reply:
Done!